generic_config_updater/patch_sorter.py

```diff
@@ -393,11 +393,23 @@ def __init__(self, path_addressing):
         self.create_only_patterns = [
             ["PORT", "*", "lanes"],
             ["LOOPBACK_INTERFACE", "*", "vrf_name"],
+            ["BGP_NEIGHBOR", "*", "holdtime"],
+            ["BGP_NEIGHBOR", "*", "keepalive"],
+            ["BGP_NEIGHBOR", "*", "name"],
+            ["BGP_NEIGHBOR", "*", "asn"],
+            ["BGP_NEIGHBOR", "*", "local_addr"],
+            ["BGP_NEIGHBOR", "*", "nhopself"],
+            ["BGP_NEIGHBOR", "*", "rrclient"],
```

Comment:
>BGP_NEIGHBOR [](http://example.com/codeflow?start=14&length=12)

Also applied to BGP_INTERNAL_NEIGHBOR, BGP_MONITORS, BGP_PEER_RANGE, BGP_VOQ_CHASSIS_NEIGHBOR ?

Reply:
Can I do this in a separate PR? I need to close the current issue.

#2029